x-pack/legacy/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -47,6 +48,15 @@ export const routes: BreadcrumbRoute[] = [
     }),
     name: RouteName.SERVICES
   },
+  {
+    exact: true,
+    path: '/settings',
+    component: ListSettings,
```

Comment:
nit: `ListSettings` >  `SettingsList`. (ListSettings sounds like either an instruction rather than a component, or a component that displays settings for a list).